internal/policy/policy.go

```diff
@@ -356,6 +359,76 @@ type Wrapper struct {
 	Kind    Kind
 }
 
+// PSActions returns unique list of actions in a policy set.
+func PSActions(ps *runtimev1.RunnablePolicySet) []string {
```

Comment:
```suggestion
func PolicySetActions(ps *runtimev1.RunnablePolicySet) []string {
```

Reply:
I named it `PSActions` due to this linter error:
```
internal/policy/policy.go:363:6: exported: func name will be used as policy.PolicySetActions by other packages, and that stutters; consider calling this SetActions (revive)
```

Maybe I should ignore the linter error since `PSActions` is a bit cryptic?

Reply:
`ListActions` and `ListPolicySetActions`?